18 ReactHotLoader/webpack.config.js

```diff
@@ -64,6 +62,8 @@ module.exports = {
 	},
 
   plugins:[
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin(),
```

Comment:
I don't know why we need this [plugin](https://github.com/webpack/docs/wiki/list-of-plugins#noerrorsplugin). It's to avoid show erros while webpack is compiling